webhook.go

```diff
@@ -97,3 +107,19 @@ func (s *WebhookServiceOp) Update(webhook Webhook) (*Webhook, error) {
 func (s *WebhookServiceOp) Delete(ID int) error {
 	return s.client.Delete(fmt.Sprintf("%s/%d.json", webhooksBasePath, ID))
 }
+
+// Verify a webhook created through the Shopify API
+func (s *WebhookServiceOp) Verify(httpRequest *http.Request) bool {
```

Comment:
Since this is not really a webhook service operation (i.e. an operation that calls a Shopify endpoint), but more of a general app-wide verification of webhooks, could you perhaps move this function to a separate file (such as a `auth.go` or even the main `goshopify.go` file) or at least make the function an extension of the `App` struct instead of `WebhookServiceOp`, similarly to how the oauth [`VerifyMessage`](https://github.com/getconversio/go-shopify/blob/4cc03b3efeb45c9d019cb2463aaf331bf8afb75c/oauth.go#L50-L59) function is created.

In other words, it would probably be fine to just change the signature of this function to something like:
```
// Verifies a webhook http request, sent by Shopify.
// The body of the request is still readable after invoking the method.
func (app App) VerifyWebhookRequest(httpRequest *http.Request) bool {
}
```

Reply:
Done!